### ISSUE TITLE
[Feature request] Fail2ban

### ISSUE THREAD

**Issue by vivienney** [QROkes/webinoly#12]
Is there any plan to implement Fail2ban for Nginx?

**Comment by QROkes** [comment]
No, Webinoly helps prevent brute force attacks by using the "limit_req" Nginx module, also wp-login pages are behind HTTP Auth by default.

**Comment by nhanledev** [comment]
The post author might want to jail the brute force in SSH or something like that. @vivienney I prefer to change your sshd port instead of relying on banning IPs.
